text/0000-innerhtml-binding.md

```diff
@@ -78,17 +85,41 @@ some content defined, like bellow:
 
 ```html
 <template>
-    <div lwc:innerHTML={myHTML()}>
+    <div lwc:inner-html={myHTML()}>
       Adding content when innerMTML is defined leads to a compilation error
     </div>
 </template>
 ```
 
+### Validity
+
+The `lwc:inner-html` attribute can be set on any tag in an LWC template with the

```

Comment:
bikeshedding on the name of the attribute, I think something like: `lwc:dangerous-inner-html` is a better name, to follow suit with our react friends :)

Reply:
That was actually my initial proposal but I got some pushback on such a long name. I'm fine either way, feel free to modify the RFC.